README.md

```diff
@@ -14,20 +14,25 @@ To use OCFL extensions you first need an OCFL client that supports the desired e
 
 Reference the OCFL specifications's description of [object extensions](https://ocfl.io/1.0/spec/#object-extensions) and [storage root extensions](https://ocfl.io/1.0/spec/#storage-root-extensions).
 
-A *root extension directory* refers to the directory named `extensions` that is located in either the storage root or an object root. An *extension directory* is an extension specific directory that is the child of a root extension directory and named using the extension's *Registered Name*. For example, `extensions/0000-example-extension` is the extension directory for the extension [0000-example-extension](docs/0000-example-extension.md).
+The OCFL storage root MAY contain a copy of an extension's specification.
 
 Each extension specification details how it should be implemented, but there are a few rules that apply to every extension.
 
-The OCFL storage root MAY contain a copy of an extension's specification.
+A *root extension directory* refers to the directory named `extensions` that is located in either the storage root or an object root. An *extension directory* is an extension specific directory that is the child of a root extension directory and SHOULD be named using the extension's *Registered Name*. For example, `extensions/0000-example-extension` is the extension directory for the extension [0000-example-extension](docs/0000-example-extension.md).
```

Comment:
Which is best?
  * SHOULD be named using the registered name
  * MUST be named either using the registered name or `init` (forward ref.)

I think I prefer the latter